Makefile

```
@@ -65,12 +65,26 @@ start-mongo:
      --name skynet-accounts-mongo-test-db \
      -p 127.0.0.1:37017:27017 \
      -e MONGO_INITDB_ROOT_USERNAME=admin \
-     -e MONGO_INITDB_ROOT_PASSWORD=ivolocalpass \
+     -e MONGO_INITDB_ROOT_PASSWORD=aO4tV5tC1oU3oQ7u \
      mongo
 
 stop-mongo:
 	docker stop skynet-accounts-mongo-test-db
 
+# start-mailslurp starts a fake mail server listening on port 2500.
+# We use that server in our integration tests.
+start-mailslurp:
+	-docker stop mailslurp
```

Should it be

```suggestion
	docker stop mailslurp
```

?

---

Maybe use the `stop-mailslurp` target, in case we ever add e.g. some cleanup to stopping

---

That's a good idea, I'll use the target. But the dash should stay - failing to clean up properly shouldn't fail the tests.

---

Oh, I had never seen that dash in Makefiles. Just looked it up. Cool

---

Yeah, I didn't know about it, either. Chris told me.